documentation/docs/developer_guide/obsolete_and_deprecated_features/obsolete_features.md

```diff
@@ -0,0 +1,37 @@
+# Obsolete features for CABLE developers #
```

Comment:
I would merge the pages for deprecated and obsolete together. Then use the table of contents on each page to navigate between the 2.

Reply:
I have merged deprecated and obsolete together.